api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/WeightLoader.kt

```diff
@@ -197,7 +199,7 @@ private fun fillConv2DVariablesFromKeras(
             }
             "bias:0" -> {
                 val biasVariableName = convBiasVarName(layerName, dim = 2)
-                val biasShape = (model.getLayer(layerName) as Conv2D).biasShapeArray
+                val biasShape = (model.getLayer(layerName) as Conv2D).biasShapeArray!!
```

Comment:
added workaround like it is `fillBatchNormVariablesFromKeras`, probably better to remove these `!!` and just don't fill layers if they don't have bias variable.